user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java

```diff
@@ -359,6 +372,34 @@ public void setCurrentTabSaveEnabled( boolean enabled ) {
     }
   }
 
+  public void setTabSaveEnabled( String frameId, boolean enabled ) {
+    PentahoTab tab = getTabByFrameId( frameId );
+    if( tab != null ) {
```

Comment:
checkstyle violation. there are probably others. please run the checkstyle ant targets and fix any issues in the files you are touching.